gonka/chain.json

```diff
@@ -0,0 +1,254 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "gonka",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Gonka",
+  "chain_type": "cosmos",
+  "chain_id": "gonka-mainnet",
+  "bech32_prefix": "gonka",
+  "daemon_name": "inferenced",
+  "node_home": "$HOME/.inference",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 1200,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ngonka",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0,
+        "high_gas_price": 0
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ngonka"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/gonka-ai/gonka",
+    "language": {
+      "type": "go",
+      "version": "1.23.2" 
+    },
+    "consensus": {
+      "type": "cometbft",
+      "version": "v0.38.17",
+      "repo": "https://github.com/cometbft/cometbft",
+      "tag": "v0.38.17"
+    },
+    "cosmwasm": {
+      "version": "0.50.0",
+      "repo": "https://github.com/CosmWasm/wasmd",
+      "tag": "v0.50.0",
+      "enabled": true
+    },
+    "sdk": {
+      "type": "cosmos",
+      "version": "v0.53.3-ps5",
+      "repo": "https://github.com/product-science/cosmos-sdk",
+      "tag": "v0.53.3-ps5"
+    },
+    "ibc": {
+      "type": "go",
+      "version": "v8.2.0",
+      "repo": "https://github.com/cosmos/ibc-go",
+      "tag": "v8.7.0",
+      "ics_enabled": [
+        "ics20-1"
+      ]
+    },
+    "binaries": {
+      "darwin/amd64": "https://github.com/gonka-ai/gonka/releases/download/release%2Fv0.2.0/inferenced-darwin-amd64.zip?checksum=sha256:740f0667d813432494d8aee65794e3e552c5a84e67c837b5e47e8daca14ae961",
+      "darwin/arm64": "https://github.com/gonka-ai/gonka/releases/download/release%2Fv0.2.0/inferenced-darwin-arm64.zip?checksum=sha256:258fdf9a9faee08202c907034084626d221097c84d317283f0e4b4c6c7aecefc1",
+      "linux/amd64": "https://github.com/gonka-ai/gonka/releases/download/release%2Fv0.2.0/inferenced-linux-amd64.zip?checksum=sha256:24d4481bee27573b5a852265cf0672e1603e405ae1f1f9fba15a7a986feca569",
+      "linux/arm64": "https://github.com/gonka-ai/gonka/releases/download/release%2Fv0.2.0/inferenced-linux-arm64.zip?checksum=sha256:37909cbd65616f9575a328f95a1fd1a8941a9e5385d116fb507c35bc686a9949"
+    },
+    "genesis": {
+      "genesis_url": "https://github.com/gonka-ai/gonka/blob/main/genesis/genesis.json"
+    }
+  },
+  "images": [
+    {
+      "image_sync": {
+        "chain_name": "gonka",
+        "base_denom": "ngonka"
```

Comment:
This image sync would otherwise be acceptable, but currently the referenced asset doesn't have any images defined